src/_utils/generators/id.ts

```diff
@@ -0,0 +1,8 @@
+export function generateId (prefix: string, length: number = 4): string {
+  let uuid = ''
+  const dictionary = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+  for (let i = 0; i < length; i++) {
+    uuid += dictionary.charAt(Math.floor(Math.random() * dictionary.length))
+  }
+  return `${prefix}-${uuid}`
+}
```

Comment:
```
import { createId } from 'seemly'
```

Reply:
No need for the file

Reply:
perfect I didn't know you had a lib for that